### PR TITLE
Support cython-like array specification

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
@pbrunet this one is ready ;-)

File: pythran/spec.py [hunk 1]
```diff
@@ -151,20 +152,19 @@ def p_type(self, p):
         '''type : term
                 | type LIST
                 | type SET
-                | type LARRAY RARRAY
-                | type LARRAY COLUMN COLUMN RARRAY
+                | type LARRAY array_indices RARRAY
                 | type COLUMN type DICT
                 | LPAREN types RPAREN'''
+        global _warned_obsolete_array_spec
```
Comment (pbrunet) [inline pythran/spec.py]: What is it done for?

**Comment by pbrunet** [comment]
LGTM, I just want to know what this global is done for :-)

File: pythran/spec.py (hunk 1 above)
Comment (serge-sans-paille) [inline pythran/spec.py]: nothing :-) At some point i added a warning to advocate for the cython
notation, then reverted everything but this line :-)

**Comment by serge-sans-paille** [comment]
@pbrunet fixed

**Comment by pbrunet** [comment]
Ok, thanks to this clarification, I notice that there is still an import too :-p. Do you prefer to merge anyway? :-)

**Comment by serge-sans-paille** [comment]
On Thu, Mar 10, 2016 at 11:11:59AM -0800, pbrunet wrote:
> Ok, thanks to this clarification, I notice that there is still an import too :-p. Do you prefer to merge anyway? :-)

I'll remove it, no need to hurry :-)

**Pull request merged**